guides/common/modules/con_overview-of-load-balancing-in-project.adoc

```diff
@@ -0,0 +1,6 @@
+[id="overview-of-load-balancing-in-project_{context}"]
+= Overview of load balancing in {Project}
+
+You can configure {ProjectServer} to use a load balancer to distribute host requests and network load across multiple {SmartProxyServers}.
```

Comment:
```suggestion
You can configure your {Project} environment to use a load balancer to distribute host requests and network load across multiple {SmartProxyServers}.
```
You don't really configure ProjectServer, so that introductory wording was a bit misleading.